packages/ez-react/src/components/Chart.tsx

```diff
@@ -76,6 +72,20 @@ export const Chart: FC<ChartProps> = ({
     [padding]
   );
 
+  const containerDimensions: Dimensions = useMemo(
+    () => ({
+      width:
+        parentDimensions?.width ||
+        dimensions?.width ||
+        defaultChartDimensions.width,
+      height:
+        parentDimensions?.height ||
+        dimensions?.height ||
```

Comment:
if `parentDimensions?.[dimension]` is `0`, then -> are we going to take `dimensions?.[dimension]` or to take this `0`? If we're going to go on with this `0` value, then the `||` operator is the wrong one, and we should use the [nullish coealescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) `??` in that case ;) But if `0` is not an acceptable value (edge-case anyway 😅 ), then this logical OR `||` is alright 👍 😄

Reply:
Let me explain it to you like this: 
- if the user provides dimensions using the `dimenions` prop, the chart will have a fixed size with the dimensions given by the user.
- if the user is wrapping the chart with this component, he doesn't have to give a fixed dimension to his chart. The chart will be responsive, so it will take the size that you are allowing it to have. 
 - if the user doesn't use this container, nor give his chart a dimension. It will have a default dimension that we have fixed to `{width: 800, height: 600}`
 I hope this is clear enough. Thank you for your comment man